docs/src/componentDocs/Grade/playground/PlaygroundPage.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import { Grade, GradeProps } from '@brightlayer-ui/react-native-components';
+import Stack from '@mui/material/Stack';
+import Box from '@mui/material/Box';
+import {
+    InputConfig,
+    PreviewComponent,
+    CodeSnippetFunction,
+    getPropsToString,
+    getPropsMapping,
+    Playground,
+} from '@brightlayer-ui/react-doc-components';
+import 'prismjs/components/prism-jsx.min';
+
+const inputConfig: InputConfig = [
+    // Required Props
+    {
+        id: 'label',
+        type: 'string',
+        typeLabel: `string`,
+        initialValue: 'A+',
+        description: 'The text shown in the circle',
+        required: true,
+        category: 'Required Props',
+    },
+    // Optional Props
+    {
+        id: 'fontColor',
+        type: 'color',
+        typeLabel: 'string',
+        description: 'The color of the text label',
+        required: false,
+        category: 'Optional Props',
+        defaultValue: 'inherit',
+        allowMuiColors: true,
+    },
+    {
+        id: 'backgroundColor',
+        type: 'color',
+        typeLabel: 'string',
+        description: 'The background color of the circle',
+        required: false,
+        category: 'Optional Props',
+        defaultValue: 'inherit',
```

Comment:
Can we add defaultValue for backgroundColor prop as per A+ grade i.e. `#198900`?